src/web/src/app/space/add/space-key-validator.ts

```diff
@@ -13,9 +14,16 @@ export class SpaceKeyValidatorDirective implements AsyncValidator {
 	constructor(private space: SpaceService) {}
 
 	validate(control: AbstractControl): Promise<ValidationErrors | null> | Observable<ValidationErrors | null> {
-		return this.space.isExists(control.value)
-			.pipe(map(res => {
-				return res ? { 'keyExists': true} : null;
-			}));
+		return timer(500).pipe(switchMap(()=>{
```

Comment:
Давай до и после оператора => добавим пробелы.
```suggestion
		return timer(500).pipe(switchMap(() => {
```

Reply:
Готово.